src/components/card_component/Card/Card.js

```diff
@@ -27,15 +27,19 @@ function Card({ characterName, house, picture }) {
 
   return (
     <section className="card">
-      <h2 className="card_subtitle">{characterName}</h2>
-      <img
-        className="character_picture"
-        src={picture}
-        alt={`Picture of ${characterName}`}
-        // Eigentlich brauchen wir 'Picture of' nicht, weil der Screenreader automatisch erkennt, dass es sich um ein Picture handelt.
-      />
-      <p className={houseColor}> House {house}</p>
-      <button>more</button>
+      <div className="picture">
+        <img
+          className="character_picture"
+          src={picture}
```

Comment:
Usually we call it `image` instead of `picture`. Also, this would better be `imageUrl` or something like that as it refers to the path.

Reply:
Changed it :pray: